src/components/Editor/Editor.jsx

```diff
@@ -118,12 +121,17 @@ class Editor extends React.Component {
   }
 
   getAnnotations = (value, updateLinting, options) => {
-  	if (!value.trim().length) {
-  		return updateLinting([]);
-  	}
+    if (!value.trim().length) {
+      return updateLinting([]);
+    }
 
-  	options.callback(value, (errors) => updateLinting(errors));
-  }
+    options.callback(value, errors => updateLinting(errors));
+  };
+
+  /**
+   * Gets the current keyMap of the editor.
+   */
+  keyMap = () => (config()._get('editor.vim') ? 'vim' : 'sublime');
```

Comment:
Please, access it using the following instead:
```js
config().editor.vim
```
`config()._get` is meant to be avoided as much as possible.

Reply:
Ah! Sorry I haven't really used the `electron-store` package however I should've inferred this due to the underscore. :)